HomeWork/src/main/java/HomeWork6/Car.java

```diff
@@ -0,0 +1,16 @@
+package HomeWork6;
+
+public class Car {
+    private void startElectricity () {
+    }
+    private void startCommand () {
+    }
+    private void startFuelSystem () {
+    }
+
+    public void start () {
```

Comment:
Let's make this program a bit more interesting. Let's print the message "Engine is started" at the end of `start()` method's body. Moreover, each method, that is called within the `start()` should also print a message, for example, "Electricity is started" etc.